thirdparty/build-thirdparty.sh

```diff
@@ -368,13 +368,8 @@ build_glog() {
     check_if_source_exist $GLOG_SOURCE
     cd $TP_SOURCE_DIR/$GLOG_SOURCE
 
-    # to generate config.guess and config.sub to support aarch64
-    rm -rf config.*
```

Comment:
Will be safe to remove this?

Reply:
change to cmake instead of autoconfig, cmake will take care everything.